src/libm/sleefsimddp.c

```diff
@@ -2407,6 +2407,10 @@ EXPORT CONST vdouble xfma(vdouble x, vdouble y, vdouble z) {
 }
 
 EXPORT CONST vdouble xsqrt_u05(vdouble d) {
+#ifdef ACCURATE_SQRT
```

Comment:
I just want to make sure that the polynomial approximation gets tested.

How about this:

```
vdouble xsqrt(vdouble x) {
#ifdef ACCURATE_SQRT
return vsqrt_vd_vd(x);
#endif
// fall back to polynomial approximation if ACCURATE_SQRT is undefined
return xsqrt_05(x);
}

vdouble xsqrt_05(vdouble x) {
  // polynomial implementation
}

// test  both xsqrt and xsqrt_05 in tester
// export only xsqrt in sleef.h
```